src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/Interaction.vb

```diff
@@ -41,6 +58,144 @@ Namespace Microsoft.VisualBasic
             Return falsePart
         End Function
 
+        Public Function Partition(ByVal Number As Long, ByVal Start As Long, ByVal [Stop] As Long, ByVal Interval As Long) As String
+            Dim Lower As Long
+            Dim Upper As Long
+            Dim NoUpper As Boolean
+            Dim NoLower As Boolean
+            Dim Buffer As StringBuilder
```

Comment:
>Buffer As StringBuilder [](start = 16, length = 23)

Note: The reference source uses `String` rather than `StringBuilder` for `Buffer`, although there are comments there to consider using `StringBuilder` instead.

Reply:
Should I rewrite to match reference source implementation or keep this way (with StringBuilder)?

Reply:
We should match the reference source as close to 100% as possible.  We don't want to try to innovate or improve perf or any such things, at least not in initial ports.

Reply:
Roger that, working on it.

Reply:
Thanks.